### ISSUE TITLE
Dropzone doesn't return successful result for images over ~1mb

### ISSUE THREAD

**Issue by nonissue** [devonbarrett/meteor-dropzone#21]
So, I think I've stumbled across a bug. If you're using images over 1mb, dropzone.js and dropzone's events (like queuecomplete or success), meteor-dropzone doesn't work. For smaller images, everything goes as planned. However, if an image is over 1mb (this is approximated, I didn't test it exactly), dropzone events are not triggered properly, even though the image may be inserted into the database correctly. 

Some of this is serious spaghetti code — the result of me spending a few hours debugging.

```javascript
  Dropzone.autoDiscover = false;
    var dropzone = new Dropzone("form#dropzone", {
      acceptedFiles: "image/gif,image/jpg,image/jpeg,image/png",
      maxFilesize: 256,
      maxThumbnailFilesize: 256,
      autoProcessQueue: true,
      addRemoveLinks: true,
      uploadMultiple: true,
      init: function() {
        var self = this;
        self.on("complete", function(file) {
          console.log("complete called");
        });
        self.on("processing", function(file) {
          console.log("processing: ", file);
        })
        self.on("success", function(file) {
          console.log("success called for file: " + file);
          ProjectImages.insert(file, function(error, result) {
            if(error) {
              alert("Error: ", error);
            } else {
              console.log("inserting image: " + result);
              var image = result;
              projectImagesArray.push(result);
            }
          });
          console.log("after: success  called for file: " + file);
        });
        self.on("addedfile", function(file) {
          console.log('file added: ' + file);
        });
        self.on("totaluploadprogress", function(progress) {
          console.log("total progress ", progress);
        });
        self.on("queuecomplete", function(progress) {
          console.log("Queue Complete.");
        });
      },
      accept: function(file, done) {
        done();
      }
    });
```

I'm using dropzone with a traditional form: 
```html
<form action="/file-upload" class="dropzone" id="dropzone"></form>
```
as I can't find any documentation on how to use the url scheme for traditional inserts.

Basically small images work and all events fire as they should. Larger images do eventually get uploaded/inserted, but things like uploadprogress only fire once and the complete/success/queuecomplete events never fire.

Let me know if this doesn't make any sense.

Edit:

Also, I'm using CollectionFS and gridfs, but it seems to occur without the database insert.

**Comment by jssolichin** [comment]
+1

**Comment by lytstephen** [comment]
+1, exact same problem, seems no issue with files under 1MB. So frustrated at the moment, spend 1+ hour trying to debug without success.

I've disabled all server side and configurations, just simple client side, and it hangs in the middle of progress, doesn't fire complete.

Any help please?

**Comment by devonbarrett** [comment]
Nice work! 

Yes, by default all Dropzone does is make a POST request to a url of your choice.

This does seem like a Dropzone specific issue rather than a Meteor one, so will close this here. (Let me know if that is wrong?)
